src/app/payments/helpers.ts

```diff
@@ -305,3 +306,31 @@ export const newCheckTwoFALimits = async ({
     walletVolume,
   })
 }
+
+export const getPriceRatioForLimits = async <
```

Comment:
this should not be in `helpers` (this file should not exists), this is related to limits

Reply:
Not sure, it's  more at the use-case level because it calls on external services (won't be able to move to domain)

Reply:
you can pass usdFromBtcMidPriceFn and MIN_SATS_FOR_PRICE_RATIO_PRECISION from app layer or should be part of price (app layer)